package.json

```diff
@@ -13,7 +13,8 @@
     "dist-all": "build -mwl --x64 --ia32",
     "pack": "build --dir",
     "dist": "build --x64 --ia32",
-    "test": "mocha test"
+    "test": "eslint . && mocha test",
```

Comment:
I removed eslint for now because I've noticed that if one runs it with old node version it goes crazy. And I suggest adding istanbul to test script.